decentraland/unity-explorer
# Fix: Test scenes are included in the build

## What does this PR change?
Fix #1972

Moved test scenes out and added them to `Tests.asmdef` to exclude from the build

## How to test the changes?

1. Check the directory of the build
2. It should not contain test scenes (see mentioned issue for screenshot and details)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
